.swiftlint.yml

```diff
@@ -2,6 +2,7 @@
 line_length: 245
 
 disabled_rules:
+  - identifier_name # Disables SwiftLint complaining about enums with associated values
```

Comment:
This got fixed in a new version of SwiftLint. If it's a local version that's running, updating should fix.

Reply:
I did update my local version of SwiftLint to 0.25.1 and was still seeing this. I'll check again